cassandane/Cassandane/TestEntity/Factory/AddressBook.pm

```diff
@@ -1,6 +1,7 @@
 package Cassandane::TestEntity::Factory::AddressBook;
 use Moo;
 
+use lib '.';
```

Comment:
```
Cassandane: add "use lib '.'" to TestEntity classes

ugh
```

I laughed, but also, this commit could use an explainer. I'm surprised it's needed